src/groups/mqb/mqbmock/mqbmock_storagemanager.cpp

```diff
@@ -0,0 +1,304 @@
+// Copyright 2021-2024 Bloomberg Finance L.P.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// mqbmock_storagemanager.cpp                                         -*-C++-*-
+#include <mqbmock_storagemanager.h>
+
+#include <mqbscm_version.h>
+
+// BDE
+#include <bsls_annotation.h>
+
+namespace BloombergLP {
+namespace mqbmock {
+
+// --------------------
+// class StorageManager
+// --------------------
+
+// CREATORS
+StorageManager::StorageManager()
+{
+    // NOTHING
+}
+
+StorageManager::~StorageManager()
+{
+    // NOTHING
+}
+
+// MANIPULATORS
+int StorageManager::start(
+    BSLS_ANNOTATION_UNUSED bsl::ostream& errorDescription)
+{
+    return 0;
+}
+
+void StorageManager::stop()
+{
+    // NOTHING
+}
+
+void StorageManager::initializeQueueKeyInfoMap(
+    BSLS_ANNOTATION_UNUSED const mqbc::ClusterState* clusterState)
```

Comment:
Alternatively you might remove the annotation and arg variable name to silence the unused variable compiler warning

Reply:
Will keep it as `BSLS_ANNOTATION_UNUSED` to be consistent with other functions.

Reply:
d